lib/zlib.js

```diff
@@ -54,8 +53,8 @@ const codes = {
 };
 
 const ckeys = Object.keys(codes);
-for (var ck = 0; ck < ckeys.length; ck++) {
-  var ckey = ckeys[ck];
+
+for (const ckey of ckeys) {
```

Comment:
Why would it matter in this case @jasnell ?

(By the way - for .. of is optimized in several cases in TurboFan, can find a reference if you'd like)